### PR TITLE
Mecanum drive

### PR DESCRIPTION
**Pull request by Rkortlandt**
Add Mecanum Drive

### CODE REVIEW

File: src/main/java/frc/robot/ChargedUp/MecanumDrive/Subsys_MecanumDrive.java [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    m_rearLeftMotor.setVoltage(volts.rearLeftVoltage);
+    m_frontRightMotor.setVoltage(volts.frontRightVoltage);
+    m_rearRightMotor.setVoltage(volts.rearRightVoltage);
+  }
+  public void zeroGyro() {
+    gyroSubSys.zeroYaw();
+  }
+
+
+  /*** Initializes Shuffleboard motor objects */
```
Comment (SeanErn) [inline src/main/java/frc/robot/ChargedUp/MecanumDrive/Subsys_MecanumDrive.java:92]: Remove this PLEASE

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -30,32 +33,19 @@
  * commands, and button mappings) should be declared here.
  */
 public class RobotContainer {
-  // The robot's subsystems and commands are defined here...
```
Comment (SeanErn) [inline src/main/java/frc/robot/RobotContainer.java:33]: Do we need these comments @chrisdeboer?

**Review by SeanErn — CHANGES_REQUESTED** [review]

**Review by SeanErn — APPROVED** [review]
Good job

**Comment by SeanErn** [comment]
still need fix

**Comment by SeanErn** [comment]
No both S capitals

File: src/main/java/frc/robot/ChargedUp/MecanumDrive/SubSys_MecanumDrive.java [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  
+  private final MecanumDrive m_MecanumDrive = new MecanumDrive(m_frontLeftMotor, m_rearLeftMotor, m_frontRightMotor, m_rearRightMotor);
+  
+  private SubSys_PigeonGyro gyroSubSys; 
+
+  public SubSys_MecanumDrive() {
+    m_frontRightMotor.setInverted(true);
+    m_frontLeftMotor.setInverted(true);
+    m_rearLeftMotor.setInverted(true);
+    m_rearRightMotor.setInverted(false);
```
Comment (SeanErn) [inline src/main/java/frc/robot/ChargedUp/MecanumDrive/SubSys_MecanumDrive.java:38]: Why is this here? It isn't inverted by default

**Review by SeanErn — CHANGES_REQUESTED** [review]

**Review by chrisdeboer — APPROVED** [review]

**Review by chrisdeboer — APPROVED** [review]

File: src/main/java/frc/robot/ChargedUp/MecanumDrive/SubSys_MecanumDrive.java (hunk 3 above)
Comment (chrisdeboer) [reply src/main/java/frc/robot/ChargedUp/MecanumDrive/SubSys_MecanumDrive.java:38]: It's okay for now

**Review by chrisdeboer — APPROVED** [review]

**Review by SeanErn — APPROVED** [review]

**Pull request merged**